aten/src/ATen/Context.cpp

```diff
@@ -147,6 +147,21 @@ void Context::setAllowTF32CuBLAS(bool b) {
   allow_tf32_cublas = b;
 }
 
+at::LinalgBackend Context::linalgPreferredBackend() const {
+  return linalg_preferred_backend;
+}
+
+void Context::setLinalgPreferredBackend(at::LinalgBackend b) {
+  linalg_preferred_backend = b;
+  if (b != at::LinalgBackend::Default) {
+    TORCH_WARN_ONCE(
+      "torch.backends.linalg.preferred is an experimental feature. "
```

Comment:
@ngimel suggests 

torch.backends.cuda.preferred_linalg_library

To capture this is CUDA only and that it's preferring a particular library

Reply:
The flag can't be changed to the first level under `torch.backends.cuda`, so it is changed to `torch.backends.cuda.linalg.preferred_library`